src/Components/test/E2ETest/Tests/JSRootComponentsTest.cs

```diff
@@ -50,7 +49,7 @@ public void CanAddAndDisposeRootComponents(bool intoBlazorUi, bool attachShadowR
 
         // They render and work
         var containerId = intoBlazorUi ? "container-rendered-by-blazor" : "root-container-1";
-        var dynamicRootContainer = Browser.FindElement(By.Id(containerId));
+        ISearchContext dynamicRootContainer = Browser.FindElement(By.Id(containerId));
```

Comment:
Just curious: What are the changes in this file about❔

Reply:
Selenium (breaking) API changes.